src/main/resources/story/reddit/CreateAccountReddit.story

```diff
@@ -0,0 +1,33 @@
+Description: Account creation
+
+Scenario: Open reddit.com main page
+Meta:
+   @test
+Given I am on the main application page
+
+Scenario: Create user account
+Meta:
+   @test
+When I refresh the page
+!-- When I decline an action in an alert with the message 'www.reddit.com wants to Show notifications'
+When I click on element located `By.xpath(//a[text()='sign up'])`
+When I switch to a frame by the xpath '//iframe[contains(@src,'actionSource')]'
+When I change context to an element by the xpath '//div[@class='Container']'
+When I enter '<email>' in a field by the xpath '//input[@id='regEmail']'
+When I click on a button with the name 'Next'
+When I select a random username
+When I change context to an element by the xpath '//div[contains(@class,'usernameStep')]'
+When I enter '<password>' in a field by the xpath '//input[@id='regPassword']'
```

Comment:
All these input fields have more friendly values under attributes 'name' or 'placeholder', so you can use them